### DISCUSSION TITLE
I have started working on this ticket.

### DISCUSSION THREAD

**Discussion by jankitrigyn** [Sunbird-Ed/Community#474] [Bugs] [answered]
whenever i am filtering the text book by using (Borad, medium and class) please see the details below
board->CBSE/NCERT
Medium->English
Class->1
on the mobile application list of books are coming, but when i am trying in web portal then i am geeting the response like ("Board is adding books") please see the attach snapshort 
<img width="910" alt="DIK-7535" src="https://user-images.githubusercontent.com/122349574/233563205-2edd53fd-8a5f-4605-b243-5dc38f4751a0.png">

**Comment by Krishnaj20** [comment, marked as answer]
@jankitrigyn -  The resolution to this is already provided in https://project-sunbird.atlassian.net/browse/ED-1748. Let us know what else are you looking for?

**Reply by Krishnaj20** [reply]
@jankitrigyn - Let us know if you had a chance to look at my query.
